app/src/main/java/com/uncookthebook/app/models/User.java

```diff
@@ -11,7 +11,7 @@
  */
 @Getter
 @ToString
-public final class User implements Model {
+public class User implements Model {
```

Comment:
why did you remove the final? I fail to see where you need to reuse it

Reply:
Because I didn't put it in the others and so I removed it. I'll add it in every Model class